src/pages/AddressInfoPage/index.tsx

```diff
@@ -183,52 +120,53 @@ const AddressInfoPage = () => {
         const price = await fetchAssetPrice('alephium')
 
         setAddressWorth(calculateAmountWorth(BigInt(balance), price))
-      } catch (error) {
-        displayError(setSnackbarMessage, error, 'Error while fetching fiat price')
+      } catch (e) {
+        console.error(e)
+        displaySnackbar({
+          text: getHumanReadableError(e, 'Error while fetching fiat worth'),
+          type: 'alert'
+        })
       }
     }
 
     getAddressWorth()
-  }, [addressBalance?.balance, setSnackbarMessage])
+  }, [addressBalance?.balance, displaySnackbar])
+
+  // Refetch TXs when less txs are found in mempool
+  useEffect(() => {
+    if (addressMempoolTransactions.length < lastKnownMempoolTxs.current.length) {
+      refetchTxList()
+    }
+    lastKnownMempoolTxs.current = addressMempoolTransactions
+  }, [addressMempoolTransactions, refetchTxList])
+
+  const totalBalance = addressBalance?.balance
+  const lockedBalance = addressBalance?.lockedBalance
+
+  const totalNbOfAssets = addressAssetIds.length + (totalBalance && BigInt(totalBalance) > 0 ? 1 : 0)
```

Comment:
This is done to take ALPH into consideration, right? So, if we only have locked ALPH and no available ALPH, we shouldn't take it into consideration for the total number of assets?